### PR TITLE
fix(breadcrumb): browsers that do not support ResizeObserver

### PR DESCRIPTION
**Pull request by RogerRosado**
#1158

Closes #

**Summary**

- summary_here

**Change List (commits, features, bugs, etc)**

- items_here

**Acceptance Test (how to verify the PR)**

- tests here

### CODE REVIEW

File: src/components/Breadcrumb/Breadcrumb.jsx [hunk 1]
```diff
@@ -60,10 +60,17 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
...
-    warning(
-      browserSupportsResizeObserver && hasOverflow,
-      'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
-    );
+    if (hasOverflow) {
+      warning(
+        browserSupportsResizeObserver,
+        'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
+      );
+    } else {
```
Comment (stuckless) [inline src/components/Breadcrumb/Breadcrumb.jsx:68]: I don't think we need this warning... really we only need the warning if hasOverflow is set to true, and we don't have a ResizeObserver.  If hasOverflow is false but the browser doesn't has a resizeobserver, we don't care.

File: src/components/Breadcrumb/Breadcrumb.jsx [hunk 2]
```diff
@@ -114,7 +121,7 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
       ref={browserSupportsResizeObserver ? breadcrumbRef : null}
       data-testid="overflow"
     >
-      {hasOverflow ? (
+      {hasOverflow && browserSupportsResizeObserver ? (
```
Comment (stuckless) [inline src/components/Breadcrumb/Breadcrumb.jsx:124]: There a several hasOverflow checks in the code, so, what I would do is right after (and outside)  the `if (__DEV__)` check I would just reassign the hasOverflow.

`hasOverflow = browserSupportsResizeObserver && hasOverflow;`

That way all the code checks for hasOverflow will be false when `browserSupportResizeObserver` is false.

Comment (RogerRosado) [reply src/components/Breadcrumb/Breadcrumb.jsx]: Implemented suggested change

File: src/components/Breadcrumb/Breadcrumb.jsx (hunk 1 above)
Comment (RogerRosado) [reply src/components/Breadcrumb/Breadcrumb.jsx]: implemented suggested change

File: src/components/Breadcrumb/Breadcrumb.jsx [hunk 3]
```diff
@@ -60,15 +60,19 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
   });
 
   if (__DEV__) {
-    warning(
-      browserSupportsResizeObserver && hasOverflow,
-      'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
-    );
+    if (hasOverflow) {
+      warning(
```
Comment (stuckless) [inline src/components/Breadcrumb/Breadcrumb.jsx:64]: I had to go looking for the documentation on `warning`.  That's one counter-intuitive function :(.  This expression is correct, but, at first , I was going to ask that check be `!browserSupportResizeObserver`, but I see the warning is thrown when the expression doesn't evaluate to `true`.

**Pull request merged**